addons/im_livechat/views/im_livechat_channel_templates.xml

```diff
@@ -62,16 +47,18 @@
                             text-align: center;
                         }
                         .main h1 {
-                            font-size: 54px;
+                            font-size: clamp(2rem, 6vw, 3.5rem);
```

Comment:
:pick: I would go a bit higher for the minimum size. It feels a bit small in mobile. What about `2.5`?